src/playground/App.js

```diff
@@ -21,6 +21,28 @@ const rulesMeta = Array.from(rules.entries()).reduce((result, [key, value]) => {
     return result;
 }, {});
 
+const fillOptionsDefaults = (options = {}) => ({
+    ...options,
+    env: {
+        es6: true,
+        ...options.env
+    },
+    parserOptions: {
+        ...options.parserOptions,
+        ecmaFeatures: {
+            ecmaVersion: "latest",
+            sourceType: "script",
+            ...options.ecmaFeatures
+        }
+    },
+    rules: options.rules ?? [...rules.entries()].reduce((result, [ruleId, rule]) => {
```

Comment:
Rules should be empty `{}` if a config without rules property was passed to the playground. Only if there's no previous config (e.g., someone opens the Playground for the first time), we're enabling recommended rules by default.

For docs examples, the rule will be enabled by a configuration comment, and it seems better not to show errors from unrelated rules.